recipes.js

```diff
@@ -1,10 +1,110 @@
-const mongoose = require('mongoose');
-const Schema   = mongoose.Schema;
-const data = require('./data.js');
-
-mongoose.connect('mongodb://localhost/recipeApp')
-  .then(() => {
-    console.log('Connected to Mongo!');
-  }).catch(err => {
-    console.error('Error connecting to mongo', err);
-  });
+const mongoose = require("mongoose");
+const express = require("express");
+const app = express();
+app.use(require('easy-livereload')());
+
+//const Schema = mongoose.Schema;
+const chalk = require("chalk");
+
+const data = require("./data.js");
+const Recipe = require("./models/recipe-model.js");
+
+mongoose
+	.connect("mongodb://localhost/recipeApp")
+	.then(() => {
+		console.log(chalk.green("Connected to Mongo! ❤️"));
+	})
+	.catch(err => {
+		console.log(chalk.red.bold("Error connecting to Mongo....... 💩", err));
+	});
+
+// set server
+app.listen(3000, () => {
+	console.log("expressive llamas in pyjamas baby");
+});
+
+// view engine and static assets
+app.use(express.static(__dirname + "/public"));
+const hbs = require("hbs");
+app.set("view engine", "hbs");
+hbs.registerPartials(__dirname + "/views/partials");
+
+// seeding from data
+Recipe.insertMany(data, function (error, docs) {
+	if (error) {
+		console.log(
+			chalk.red.bold(
+				"Could not bulk import data from existing collection 💩",
+				error
+			)
+		);
+	} else {
+		console.log(chalk.green.bold("Bulk import went smooth", docs));
+	}
+});
+
+// udptating one of them
+Recipe.findOneAndUpdate({
+		title: "Rigatoni alla Genovese"
+	}, {
+		$set: {
+			duration: 100
+		}
+	},
+	(err, doc) => {
+		if (err) {
+			console.log("Something wrong when updating data!");
+		}
+
+		console.log(chalk.magenta.bold("Recipe successfully updated to:"));
+		console.log(chalk.white.dim(doc));
+	}
+);
+
+Recipe.deleteOne({
```

Comment:
You could use the `findByIdAndUpdate()` function for this in case there is two recipes with the same name it would delete both. Same for the update function.